api_gateway/src/main/kotlin/pl/edu/pg/rsww/apigateway/RoutingController.kt

```diff
@@ -27,6 +25,36 @@ public class RoutingController(
         return "Hello, I'm alive!"
     }
 
+    @RequestMapping("/api/mocked/{*path}")
+    fun mock(
+        @PathVariable path: String,
+        @RequestParam params: Map<String, String>,
+    ): ResponseEntity<String> {
+        val builder = ResponseEntity.status(200)
+        if (path.contains("get_trips")) {
+            val n: Int = params["n"]?.toInt() ?: 0
+            val next = n + 1
+
+            return builder.body(
+                """{
```

Comment:
Not that I would be complaining but did we end up deciding on using JSON in the responses and templating on the frontend's side or does this deviate for other reasons?

Reply:
Mocking json answer made it easier to iterate over, but micro-service returning materialized template would be equally valid